src/main/java/com/browserstack/automate/ci/common/proxysettings/JenkinsProxySettings.java

```diff
@@ -0,0 +1,77 @@
+package com.browserstack.automate.ci.common.proxysettings;
+
+import hudson.ProxyConfiguration;
+import jenkins.model.Jenkins;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+
+public class JenkinsProxySettings {
+
+    static final ProxyConfiguration jenkinsProxy = Jenkins.getInstanceOrNull() != null ? Jenkins.getInstanceOrNull().proxy : null;
```

Comment:
```suggestion
    private static final ProxyConfiguration jenkinsProxy = Jenkins.getInstanceOrNull() != null ? Jenkins.getInstanceOrNull().proxy : null;
```

Please define correct accessSpecifiers wherever necessary.